### PR TITLE
Don't send usage information on Chrome infra bots

### PR DESCRIPTION
**Pull request by eseidelGoogle**
There are a bunch of properties I could check, but this one seemed easiest.

@devoncarew @abarth

### CODE REVIEW

**Comment by sethladd** [comment]
yay! Good find.

**Comment by sethladd** [comment]
lgtm

**Comment by eseidelGoogle** [comment]
I was watching an old talk on understanding your data https://www.youtube.com/watch?v=-3dw09N5_Aw which made me want to look at our analytics, and this popped up.

Thx for the review.

**Comment by sethladd** [comment]
Really appreciate it. Devon and I were scratching our heads wondering where the other bot was. :)

File: packages/flutter_tools/lib/src/base/utils.dart [hunk 1]
```diff
@@ -13,7 +13,9 @@ bool get isRunningOnBot {
   // https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
   return
     Platform.environment['TRAVIS'] == 'true' ||
-    Platform.environment['CONTINUOUS_INTEGRATION'] == 'true';
+    Platform.environment['CONTINUOUS_INTEGRATION'] == 'true' ||
+  // CHORME_HEADLESS is one property set on Flutter's Chrome Infra bots.
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/base/utils.dart]: `CHORME_HEADLESS` or `CHROME_HEADLESS`?

**Comment by devoncarew** [comment]
Thanks! Looking forward to not having bots in our data anymore :)

**Comment by eseidelGoogle** [comment]
This didn't make Ashburn, VA go away.

**Comment by devoncarew** [comment]
Did you see my comment about a likely misspelling? `CHORME_HEADLESS ` vs `CHROME_HEADLESS`

**Comment by sethladd** [comment]
And that's why you don't have me review PRs like this :)

File: packages/flutter_tools/lib/src/base/utils.dart (hunk 1 above)
Comment (eseidelGoogle) [inline packages/flutter_tools/lib/src/base/utils.dart]: Thanks for fixing in #4364

**Comment by eseidelGoogle** [comment]
And why I shouldn't write PRs in the first place. :/  Thanks for fixing (faster than I could get around to it).

**Pull request merged**